src/main/scala/org/apache/mesos/chronos/notification/HttpClient.scala

```diff
@@ -0,0 +1,93 @@
+package org.apache.mesos.chronos.notification
+
+import java.io.{DataOutputStream, StringWriter}
+import java.net.{HttpURLConnection, URL}
+import java.util.logging.Logger
+
+import org.apache.commons.codec.binary.Base64
+
+import org.apache.mesos.chronos.scheduler.jobs.BaseJob
+import com.fasterxml.jackson.core.JsonFactory
+
+class HttpClient(val endpointUrl: String, 
+                 val credentials: Option[String]) extends NotificationClient {
+
+  private[this] val log = Logger.getLogger(getClass.getName)
+
+  def sendNotification(job: BaseJob, to: String, subject: String, message: Option[String]) {
+
+    val jsonBuffer = new StringWriter
+    val factory = new JsonFactory()
+    val generator = factory.createGenerator(jsonBuffer)
```

Comment:
Hi @brndnmtthws 
I borrowed the JSON generation code from https://github.com/mesos/chronos/blob/master/src/main/scala/org/apache/mesos/chronos/notification/SlackClient.scala.
Do you have plans to change that as well?